include/irr/asset/IRenderpassIndependentPipeline.h

```diff
@@ -112,17 +132,17 @@ struct SRasterizationParams
     uint32_t sampleMask[2];
     E_COMPARE_OP depthCompareOp;
     float minSampleShading;
-    bool depthClampEnable;
-    bool rasterizerDiscard;
-    bool fronFaceIsCCW;
-    bool depthBiasEnable;
-    bool sampleShadingEnable;
-    bool alphaToCoverageEnable;
-    bool alphaToOneEnable;
-    bool depthTestEnable;
-    bool depthWriteEnable;
-    bool depthBoundsTestEnable;
-    bool stencilTestEnable;
+    uint16_t depthClampEnable : 1;
```

Comment:
anonymouse struct and serialize/deserialize methods

Reply:
resolved

Reply:
>serialize/deserialize methods

will be done when doing BAW format update